docs/manual/manual_safety_systems.rst

```diff
@@ -0,0 +1,255 @@
+
+1. Slew limiting disabled
+
+.. image:: ../_static/manual_images/safety/slew_rate_off.png
+    :align: center
+    :scale: 80%
+
+2. Slew limit enabled at 10V/s
+
+.. image:: ../_static/manual_images/safety/slew_rate_10vps.png
+    :align: center
+    :scale: 80%
+
+3. Slew limit enabled at 5V/s
+
+.. image:: ../_static/manual_images/safety/slew_rate_5vps.png
+    :align: center
+    :scale: 80%
+
+4. Slew limit enabled at 2.5V/s
+
+.. image:: ../_static/manual_images/safety/slew_rate_2_5vps.png
+    :align: center
+    :scale: 80%
+
+5. Slew limit enabled at 1V/s
+
+.. image:: ../_static/manual_images/safety/slew_rate_1.png
+    :align: center
+    :scale: 80%
+
+As you can see, the slew limit parameter helps protect against sudden changes in commanded voltage. This can be especially useful in :ref:`noisy/vibration heavy 
+environments <vibration_and_jittering>`. The amount of slew filtering that is necessary (if any) is completely application specific, and will require manual tuning.
+
+Brushless Drive Current Limiter
+====================================
+:ref:`Brushless Drive's <brushless_drive>` ``motor_I_max`` defines the maximum allowable drive current. In general, this current limit protects your module against extreme current surges (e.g. step commands and stalls), and ensures that the current moving through the motor is always in a safe region.
+This behavior is illustrated in the examples below. The first is a module with a ``motor_I_max`` of 50A. It is given a step to 5V, then after 5 seconds is stepped back down to a 0V control.
+
+.. note::
+    ``motor_I_max`` defines the maximum allowable motor current, and not supply current. These values are not necessarily the same.
+
+.. image:: ../_static/manual_images/safety/current_limit_50.png
+    :align: center
+    :scale: 80%
+
+Now, we set the current limit to 0.5A. You will see that the motor's rate of change is much slower. As mentioned above, this is meant to decrease the likelihood of dangerous currents as the module steps between velocities.
+
+.. image:: ../_static/manual_images/safety/current_limit_0_5.png
+    :align: center
+    :scale: 80%
+
+The biggest difference between this current limit and :ref:`slew_rate` is that the slew rate parameter defines an absolute linear relation between the control voltage applied and 
+the time it takes to get there. This current limit is not meant to limit the rate of change of commanded voltages, it is meant to protect against all high current events. Suppose 
+your module is spinning, but is caught in a tree and can no longer spin. Assuming that your flight controller continues to send commands, your module will continue to 
+attempt spinning, but if it cannot, then it will end up with a surge of current going through its coils. In this case, ``motor_I_max`` would protect the module by 
+forcing it to stop trying to spin.
```

Comment:
It actually won't make it not try to spin necessarily, it will just limit the amount of current that is being asked for by limiting the drive voltage. If you pulled it off that tree, it would start spinning again once nothing was blocking it.